jenkins/e2e-image/Dockerfile

```diff
@@ -15,57 +15,32 @@
 # This file creates a build environment for building and running kubernetes
 # unit and integration tests
 
-FROM gcr.io/k8s-testimages/kubekins-test:1.7-v20170713-c28e0556
-MAINTAINER  Erick Fejta <fejta@google.com>
+FROM gcr.io/k8s-testimages/bootstrap:v20171004-2e7545c1
+MAINTAINER  Sen Lu <senlu@google.com>
 
-# https://github.com/GoogleCloudPlatform/gsutil/issues/446 for python-openssl
-RUN apt-get update && apt-get install -y --no-install-recommends \
-    pkg-config \
-    zip \
-    unzip \
-    xz-utils \
-    zlib1g-dev \
-    python-openssl \
-    && apt-get clean
+# install go
+ARG GO_VERSION
+ENV GO_TARBALL "go${GO_VERSION}.linux-amd64.tar.gz"
+RUN wget -q "https://storage.googleapis.com/golang/${GO_TARBALL}" && \
+    tar xzf "${GO_TARBALL}" -C /usr/local && \
+    rm "${GO_TARBALL}"
```

Comment:
did we give-up on checksumming it?
https://github.com/docker-library/golang/pull/53/files

Reply:
I don't see we do checksum for other packages though. I can backfill it.